packages/calcite-components/src/components/flow/flow.e2e.ts

```diff
@@ -116,6 +116,27 @@ describe("calcite-flow", () => {
       expect(mockCallBack).toHaveBeenCalledTimes(1);
     });
 
+    it("should handle rejected 'beforeBack' promise'", async () => {
+      const page = await newE2EPage();
+
+      const mockCallBack = jest.fn().mockReturnValue(() => Promise.reject());
+      await page.exposeFunction("beforeBack", mockCallBack);
+
+      await page.setContent(`<calcite-flow><calcite-flow-item></calcite-flow-item></calcite-flow>`);
+
+      await page.$eval(
+        "calcite-flow-item",
+        (elm: HTMLCalciteFlowItemElement) =>
+          (elm.beforeBack = (window as typeof window & Pick<typeof elm, "beforeBack">).beforeBack)
+      );
+
+      const flow = await page.find("calcite-flow");
+
+      await flow.callMethod("back");
+
+      expect(mockCallBack).toHaveBeenCalledTimes(1);
+    });
```

Comment:
Can you add an assertion here to catch that the flow item wasn't 'popped'?

Reply:
done. Used new test because the mock function was messing it up.